Reduce setuptools requirement until [1] is resolved. Seems to work fine
with 63.x.

[1] https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=270358

--- pyproject.toml.orig	2025-05-16 11:58:37 UTC
+++ pyproject.toml
@@ -1,7 +1,7 @@
 # The project configuration for sip.
 
 [build-system]
-requires = ["setuptools>=77", "setuptools_scm>=8"]
+requires = ["setuptools>=63", "setuptools_scm>=8"]
 build-backend = "setuptools.build_meta"
 
 [tool.setuptools_scm]
@@ -12,10 +12,10 @@ urls.homepage = "https://github.com/Python-SIP/sip"
 description = "A Python bindings generator for C/C++ libraries"
 readme = "README.md"
 urls.homepage = "https://github.com/Python-SIP/sip"
-dependencies = ["packaging>=24.2", "setuptools>=75.8.1", "tomli; python_version<'3.11'"]
+dependencies = ["packaging>=24.2", "setuptools>=63", "tomli; python_version<'3.11'"]
 requires-python = ">=3.9"
-license = "BSD-2-Clause"
-license-files = ["LICENSE"]
+license = {file = "LICENSE"}
+classifiers = ["License :: OSI Approved :: BSD License"]
 dynamic = ["version"]
 
 [[project.authors]]
